lib/scopes.js

```diff
@@ -0,0 +1,29 @@
+var SCOPES = {
+  user: [
```

Comment:
lets add `avatar` and `id` for now... I'd actually love to ditch those `is*` booleans, and I'm not sure how much sense `prefLocale` makes right now. We can always add it later